test/UnitTests/MSTestAdapter.PlatformServices.UnitTests/Execution/TestClassInfoTests.cs

```diff
@@ -16,9 +16,9 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.UnitTests.Execution;
 
 public class TestClassInfoTests : TestContainer
 {
-    private sealed class TestTestCoxtextImpl : ITestContext
+    private sealed class TestContextImpl : ITestContext
```

Comment:
I don't think it's a typo. The name is indicating that this instance is an implementation of "TestContext" but is used for testing purposes only.

It's like having `class TestComponent : IComponent`, so you prefix the component being tested with `Test`. Here the component is `TestContext` and we are creating a fake test implementation of it.